pom.xml

```diff
@@ -130,16 +130,12 @@
         </pluginManagement>
         <plugins>
             <plugin>
-                <groupId>org.wso2.siddhi</groupId>
-                <artifactId>siddhi-doc-gen</artifactId>
-                <version>${siddhi.version}</version>
-                <executions>
-                    <execution>
-                        <goals>
-                            <goal>generate-md-docs</goal>
-                        </goals>
-                    </execution>
-                </executions>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-release-plugin</artifactId>
```

Comment:
Can you remove the duplicated plugin that doesn't run the profile -Pdocumentation-deploy

Reply:
Removed the duplicate plugin